### PR TITLE
beautify code on files in fa subdir

### PR DESCRIPTION
**Pull request by GrotheFAF**
mostly blank lines and comment corrections
 lowercase locally used variables and functions
 (no external references were harmed during the making of this commit)

Initial PR:

- [ ] PR branch should be named `issuenum`-`fix`/`feature`/`cleanup`-`description`
- [ ] Code is split into logical commits
- [ ] Code has tests
- [ ] Final commit includes "Fixes #issue" in commit message

When all builds pass and a maintainer is happy with your PR, the "ready" label will be applied. Please complete these tasks then:

- [ ] Rebase onto develop
- [ ] Add changelog entry
- [ ] Remove this entire template section

### CODE REVIEW

File: src/fa/replay.py [hunk 1]
```diff
@@ -14,26 +13,28 @@
...
+    """
     logger.info("fa.exe.replay(" + str(source) + ", detach = " + str(detach))
 
     if fa.instance.available():
         version = None
         featured_mod_versions = None
         arg_string = None
         replay_id = None
+        mod = None
+        map = None
```
Comment (Wesmania) [inline src/fa/replay.py]: 'map' is a Python keyword, we should avoid that.

**Comment by Wesmania** [comment]
I like this. Do we have coding standards for underscores / camelcase? It might be a good idea to set some up.

File: src/fa/replay.py (hunk 1 above)
Comment (GrotheFAF) [reply src/fa/replay.py]: oh, missed that  .. commit comin'.

**Comment by GrotheFAF** [comment]
This is after python rules in PyCharm:
camels and 2 blank lines for classes, everthing else is lowercase and max. 1 blank line.
(Qt and other Libs are still Pascal-ish and PyCharm is fine with it.)

**Comment by GrotheFAF** [comment]
and this will not be processed too.

**Pull request closed without merging**